packages/matterbridge/build.sh

```diff
@@ -3,7 +3,7 @@ TERMUX_PKG_DESCRIPTION="A simple chat bridge"
 TERMUX_PKG_LICENSE="Apache-2.0"
 TERMUX_PKG_MAINTAINER="@termux"
 TERMUX_PKG_VERSION="1.26.0"
-TERMUX_PKG_REVISION=2
+TERMUX_PKG_REVISION=3
```

Comment:
Wait, what changed?
You only bumped `TERMUX_PKG_REVISION`, with no changes to anything else.
Won't this package be identical to `1.26.0-2`?

Reply:
A revbump can't fix that.
It only rebuilds the package (usually to apply new patches on top of the existing version)
But a revbump alone does not change the built package in any way.